docs/modules/kubernetes/pages/kubernetes-auto-discovery.adoc

```diff
@@ -103,6 +103,8 @@ NOTE: If you don't specify `service-name` and fall back to a `namespace` only di
 * `pod-label-name`, `pod-label-value`: pod label and value used to tag Pods, which together form the Hazelcast cluster. These properties can support multiple comma-separated values. For example: "label-1,label-2". You must use the same number of elements in `pod-label-name` as `pod-label-value`. 
 * `resolve-not-ready-addresses`: if set to `true`, it checks also the addresses of Pods which are not ready; `true` by default
 * `expose-externally`: if set to `true`, it fails fast if an external address cannot be found for each member; if set to `false`, it does not check for external member addresses; by default it tries to resolve external addresses but fails silently
++
+NOTE: If your service type is `NodePort`, there might be connectivity problems from outside to the Kubernetes cluster. Therefore, it's crucial to verify that NodePort IPs are externally accessible.
```

Comment:
@semihbkgr , do we need to mention about how to verify, or is that a trivial process for the users?

Reply:
It is easy to verify whether the NodePort is accessible outside the Kubernetes network. However, the network configuration (firewall, network rules, policies, etc.) of the Kubernetes nodes can vary significantly depending on the cloud provisioner or services you are using.